config/web.php

```diff
@@ -13,4 +15,6 @@
             ->language($params['app']['language'])
             ->name($params['app']['name']);
     },
+
+    ValidatorFactoryInterface::class => ValidatorFactory::class,
```

Comment:
May it be declared directly in `yiisoft/validator`?

Reply:
> May it be declared directly in `yiisoft/validator`?

I think that this should not be in the validator, this is an independent package from yii, and it does not know anything about composere-plugin, and without composer-plugin this will not work.